tests/resolution/test_deadlock_safeguards.py

```
@@ -77,7 +73,7 @@ async def test_raises_sync_usage_error_with_prefilled_exception(
     [(0.1, 0.2), (0.2, 0.1)],
     ids=["completes", "times-out"],
 )
-async def test_from_separate_thread(method: str, coro_sleep: float, timeout: float) -> None:
+async def test_from_separate_thread(*, method: str, coro_sleep: float, timeout: float) -> None:
```

Ruff (0.15.10)

[warning] Async function definition with a `timeout` parameter

(ASYNC109)